src/CompilerInterrupt.cpp

```diff
@@ -4851,7 +4851,7 @@ namespace {
 
       DT->recalculate(*F);
       PDT->recalculate(*F);
-      BPI->calculate(*F, *LI);
+      BPI->calculate(*F, *LI, nullptr, DT, PDT);
```

Comment:
Let's support both llvm-9 & llvm-12 & use a preprocessor flag like "#ifdef LLVM12" & then set the flag in the makefile by checking if llvm-12 is present in the system or not.

Reply:
Updated in 94a352f.